cli/organization.go

```diff
@@ -0,0 +1,86 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/cli/cliui"
+	"github.com/coder/coder/v2/codersdk"
+)
+
+func (r *RootCmd) organizations() *clibase.Cmd {
+	cmd := &clibase.Cmd{
+		Annotations: workspaceCommand,
+		Use:         "organizations [subcommand]",
+		Short:       "Organization related commands",
+		Aliases:     []string{"organization", "org", "orgs"},
+		Hidden:      true, // Hidden until these commands are complete.
+		Handler: func(inv *clibase.Invocation) error {
+			return inv.Command.HelpHandler(inv)
+		},
+		Children: []*clibase.Cmd{
+			r.currentOrganization(),
+		},
+	}
+
+	cmd.Options = clibase.OptionSet{}
+	return cmd
+}
+
+func (r *RootCmd) currentOrganization() *clibase.Cmd {
+	var (
+		client    = new(codersdk.Client)
+		formatter = cliui.NewOutputFormatter(
+			cliui.ChangeFormatterData(cliui.TextFormat(), func(data any) (any, error) {
+				typed, ok := data.([]codersdk.Organization)
+				if !ok {
+					// This should never happen
+					return "", fmt.Errorf("expected []Organization, got %T", data)
+				}
+				if len(typed) != 1 {
+					return "", fmt.Errorf("expected 1 organization, got %d", len(typed))
+				}
+				return fmt.Sprintf("Current organization: %s (%s)\n", typed[0].Name, typed[0].ID.String()), nil
+			}),
+			cliui.TableFormat([]codersdk.Organization{}, []string{"id", "name", "default"}),
+			cliui.JSONFormat(),
+		)
+		onlyID = false
+	)
+	cmd := &clibase.Cmd{
+		Use:   "current",
```

Comment:
Suggestion: `s/current/show/`.

This matches better what we have in the cli already, `coder show <workspace>`, `coder users show <me>`, and now `coder org show [current|me|uuid]`. I'm thinking the arg can be optional.

Reply:
That is a good idea. If no arg is provided, I guess I could default to `current`?

Reply:
Yep, that sounds good 👍🏻

Reply:
All the output formats work as expected.

```
$ coder organizations show me
Organizations you are a member of:
        admin (653e414b-9eb7-41cb-b11c-38cf90b29db3)
        steven-org (9616b384-38bb-4284-861b-8db4fc337d6d)
        bia-org (75e6295c-0bfb-4195-8898-b9388879dafb)
```